src/zc/form/field.py

```diff
@@ -291,7 +286,7 @@ def __init__(self, may_be_equal=True, decreasing=False):
     def __call__(self, field, value):
         # can assume that len(value) == len(field.fields)
         last = None
-        for v, f in map(None, value, field.fields):
+        for v, f in zip(value, field.fields):
```

Comment:
Another possible behavioral change.

Reply:
Two lines earlier, the assumption of equal length is made.